src/styles/calendar.less

```diff
@@ -0,0 +1,146 @@
+// -------- CALENDAR COMPONENT, MODAL STYLED AT BOTTOM --------
+
+.ant-btn {
+  background-color: #fcdfaa !important;
+  position: absolute !important;
+  right: 5%;
+  top: 7.8%;
+  width: fit-content;
+  padding: 0 5% !important;
+  color: #f27712 !important;
+  font-weight: bold !important;
+}
+
+.ant-picker-calendar-header {
+  justify-content: flex-start !important;
+  margin: 0 2% !important;
+  div {
+    background-color: #f1c461 !important;
+    color: #f27712 !important;
+  }
+}
+
+.ant-radio-button-wrapper {
+  background-color: #fcdfaa !important;
+  color: #f27712 !important;
+}
+.ant-radio-button-wrapper-checked {
+  background-color: #f1c461 !important;
+  color: white !important;
+}
+
+thead {
+  background-color: #fcdfaa;
+}
+
+th {
+  padding: 2rem !important;
+}
+
+thead tr th {
+  color: #f27712 !important;
+}
+
+ul {
+  background-color: transparent !important;
+}
+
+.ant-badge-not-a-wrapper {
+  text-align: center;
+  margin-top: 5% !important;
+  padding: 4px 16px !important;
+  width: 100% !important;
+  .ant-badge-status-text {
+    color: black !important;
+    margin-left: 0 !important;
+  }
+}
+.ant-badge-not-a-wrapper:nth-of-type(1) {
+  margin-top: 0 !important;
+}
+
+.success-class-event,
+.warning-class-event,
+.error-class-event {
+  border-radius: 9999px !important;
+}
+
+.success-class-event {
+  background-color: rgba(0, 202, 183, 0.25) !important;
+}
+
+.warning-class-event {
+  background-color: #faad14 !important;
+}
+
+.error-class-event {
```

Comment:
Nice job on adding different colors for error event, warning, and success class event to differentiate one another.